easybuild/easyconfigs/p/pigz/pigz-2.4_Makefile.patch

```diff
@@ -0,0 +1,19 @@
+diff -Nru pigz-2.4.orig/Makefile pigz-2.4/Makefile
```

Comment:
@jdemeul Please include a short description above this line + mention author (and then fix the checksum in the easyconfig)